src/registration/Register.tsx

```diff
@@ -0,0 +1,223 @@
+            inputStyle={{ display: 'block' }}
+            label="First Name"
+          />
+        </div>
+        <div style={{ width: '1rem' }} />
+        <div style={{ lineHeight: '170%' }}>
+          <LabelledTextInput
+            value={familyName}
+            onChange={setFamilyName}
+            inputStyle={{ display: 'block' }}
+            label="Last Name"
+          />
+        </div>
+      </div>
+      <div style={{ lineHeight: '170%' }}>
+        <LabelledTextInput
+          value={email}
+          required
+          onChange={setEmail}
+          labelStyle={{ display: 'block', marginTop: '2rem' }}
+          inputStyle={{ width: '66ex' }}
+          label="Contact Email for Notifications"
+        />
+      </div>
+      <div style={{ lineHeight: '170%' }}>
+        <LabelledTextInput
+          value={institute}
+          required={partOfOrganization}
+          disabled={!partOfOrganization}
+          onChange={setInstitute}
+          inputStyle={{ width: '66ex' }}
+          label="Organization"
+        />
+      </div>
+      <div style={{ lineHeight: '170%', marginTop: '0.25rem' }}>
+        <LabeledCheckbox
+          checked={!partOfOrganization}
+          onChange={() => setPartOfOrganization(!partOfOrganization)}
+          disabled={false}
+        >
+          <span style={{ marginLeft: '0.25rem' }}>I am not a part of an organization</span>
+        </LabeledCheckbox>
+      </div>
+      <div style={{ display: 'flex' }}>
+        <div style={{ lineHeight: '170%' }}>
+          <LabelledTextInput
+            value={department}
+            required={partOfOrganization}
+            disabled={!partOfOrganization}
+            onChange={setDepartment}
+            labelStyle={{ display: 'block' }}
+            label="Department"
+          />
+        </div>
+        <div style={{ width: '1rem' }} />
+        <div style={{ lineHeight: '170%' }}>
+          <LabelledTextInput
+            value={title}
+            required={partOfOrganization}
+            disabled={!partOfOrganization}
+            onChange={setTitle}
+            label="Title"
+            labelStyle={{ display: 'block' }}
+          />
+        </div>
+      </div>
+      <h3 style={{ marginTop: '2rem' }}>I am most interested in using Terra to (Check all that apply):</h3>
+      <CheckboxLine>
+        {_.map(
+          (title: string) => {
+            return (
+              <InterestInTerraCheckbox
+                title={title}
+                interestInTerra={interestInTerra}
+                setFunc={setInterestInTerra}
+                key={_.uniqueId('interest_')}
+              />
+            );
+          },
+          [
+            'Collaborate with individuals within my organization',
+            'Collaborate with individuals outside of my organization',
+            'Access data',
+            'Manage datasets',
+            'Launch workflows',
+            'Complete interactive analyses',
+            'Build Tools',
```

Comment:
This entry's capitalization is inconsistent with the others.

```suggestion
            'Build tools',
```